### DISCUSSION TITLE
Error when updating system components and applications

### DISCUSSION THREAD

**Discussion by ArtemLapys** [elementary/appcenter#1934] [General]
Hello, for what week there has been a problem when trying to install updates. Can you tell me what to do? Multitasking animations and desktop switching also continue to lag.

> Error while installing package: attempt to overwrite "/usr/share/metainfo/granite.appdata.xml ", which is already available in the package libgranite-common 6.2.0+r2223+pkg122~ubuntu6.1

**Comment by lenemter** [comment]
There is a issue with the same error: #1649 

One of the comments suggests a solution: execute this command in terminal:

```
sudo apt-get install -f
```

**Reply by ArtemLapys** [reply]
I've seen this answer. It didn't help me, because I installed it through the "Application Center". I deleted some libin*** (I don't remember the exact name) and elementary-tweaks. Also removed all dependencies. As a result, there were no more mistakes. The question is, why does the zoom on the touchpad not work with two fingers? Is it possible to configure it somehow? Also set up a two-finger gesture - "back"?

**Reply by lenemter** [reply]
> The question is, why does the zoom on the touchpad not work with two fingers? Is it possible to configure it somehow? Also set up a two-finger gesture - "back"?

It's not possible out of the box, however, there are [Touchégg](https://github.com/JoseExpposito/touchegg) and [Touché](https://github.com/JoseExpposito/touche), which can bring some additional non one-to-one gestures. I haven't used them personally, so can't help any more
